server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -675,6 +677,8 @@ public void apply(Settings value, Settings current, Settings previous) {
             IndicesService.CLUSTER_REMOTE_TRANSLOG_REPOSITORY_SETTING
         ),
         List.of(FeatureFlags.CONCURRENT_SEGMENT_SEARCH),
-        List.of(SearchService.CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING)
+        List.of(SearchService.CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING),
+        List.of(FeatureFlags.TELEMETRY),
```

Comment:
We should probably move it off to `TelemetrySettings`, this is not a cluster setting: 
```
 public static final Map<List<String>, List<Setting>> FEATURE_FLAGGED_TELEMETRY_SETTINGS = Map.of(

```